.nvmrc

```diff
@@ -0,0 +1 @@
+lts/carbon
```

Comment:
We can drop this now - we've moved everything in the project up to v12 (erbium) and have `.nvmrc` files under `client` and `server`

Reply:
Oh, where was this? I didn't have it after a rebase a week ago, new change?

Reply:
About an hour ago – it's very cutting edge 😃

Reply:
I removed my nvmrc, switched back to 12 localy